gems/pending/appliance_console.rb

```diff
@@ -131,13 +132,12 @@ module ApplianceConsole
         summary_entry("Search Order", order),
         summary_entry("MAC Address", mac),
         summary_entry("Timezone", timezone),
-        summary_entry("Local Database", ApplianceConsole::Utilities.pg_status),
-        summary_entry("#{I18n.t("product.name")} Database",
-                      configured ? "postgres @ #{dbhost || "localhost"}" : "not configured"),
-        summary_entry("Database/Region", configured ? "#{database} / #{region || 0}" : "not configured"),
+        summary_entry("Local Database Server", PostgresAdmin.local_server_status),
+        summary_entry("#{I18n.t("product.name")} Server", evm_running ? "running" : "not running"),
+        summary_entry("#{I18n.t("product.name")} Database", dbhost || "not configured"),
+        summary_entry("Database/Region", database ? "#{database} / #{region.to_i}" : "not configured"),
         summary_entry("External Auth", ExternalHttpdAuthentication.config_status),
         summary_entry("#{I18n.t("product.name")} Version", version),
-        summary_entry("#{I18n.t("product.name")} Console", configured ? "https://#{ip}" : "not configured")
```

Comment:
Oops clobbered you're old comment here @jrafanie 

I figured this was a bit misleading because the webpage is only valid if you have the UI role, it really has nothing to do with if the database is configured or not.

But either way, we're printing almost exactly the same thing here https://github.com/ManageIQ/manageiq/pull/10354/files#diff-16bd2bb3c0abfe4c0381896614e7b987R104